src/app/static/vue.config.js

```diff
@@ -35,5 +37,12 @@ module.exports = defineConfig({
             hints: false
         },
         devtool: 'eval-source-map',
+        plugins: [
+            new webpack.DefinePlugin({
+                // Vue CLI is in maintenance mode, and probably won't this PR to fix this in their tooling
```

Comment:
not sure i fully understand this comment, do you mean you want that PR in?

Reply:
o lol no, I will fix the English!